types/networks/Cargo.toml

```
@@ -10,6 +10,7 @@ clock = { package = "fil_clock", path = "../../node/clock", version = "0.1" }
 lazy_static = "1.4"
 beacon = { path = "../../blockchain/beacon" }
 serde_json = "1.0"
-
+cid = { package = "forest_cid", version = "0.3" }
+ipld_blockstore = { path = "../../ipld/blockstore" }
```

don't use the path here, it will cause issues if the patch isn't applied or when things are released. The only reason this compiles is because of the patch to switch all to local for easier development
```suggestion
ipld_blockstore = "0.1"
```